extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -200,6 +200,23 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	public async openNewBook(): Promise<void> {
+		const allFilesFilter = localize('allFiles', "All Files");
```

Comment:
I'd move this into the localizedConstants file instead (same with all the other localize calls here). This has a couple of benefits :

1. Lets you easily see what other strings already exist which can help reduce duplication
1. Reduces the number of times the localize function is called - there's no reason to redo the lookup every time the function is called